dev/packages/example/log-0.9.0/dataset/log/manifest.yml

```diff
@@ -16,3 +16,12 @@ streams:
         type: text
         multi: true
 
+      - name: dataset
+        required: true
+        default: generic
```

Comment:
I think it's crashing because it's set as `multi: true` but the default value doesn't seem to be an array

Reply:
multi is not needed here. I removed it.

Reply:
+1 on having validation for it.